basic/airdrop-with-fixed-tokens-by-recipient/src/lib.rs

```diff
@@ -0,0 +1,68 @@
+use scrypto::prelude::*;
+
+
+blueprint! {
+
+    struct AirdropWithFixedTokensByRecipient {
```

Comment:
This example looks too much like this one: https://github.com/radixdlt/community-scrypto/tree/main/basic/airdrop.
Would you be able to make it more different ? For example, instead of directly sending tokens to the users, allow them to withdraw the tokens by specifying their badges.

Reply:
Hello Clement, 
Thanks for your review. 
Indeed, the 2 implementations are very close.
I will carry out the implementation you suggest.

Reply:
@cbisaillon I just updated the source code. could you check please ?